### DISCUSSION TITLE
Whether THEA server can handle multiple people request or dedicated server for each user?

### DISCUSSION THREAD

**Discussion by midhun8454** [eclipse-glsp/glsp#433] [Q&A] [answered]
Since man persons will be using this , do THEA server can handle multiple request or dedicated server for each request ?
if so , can you please explain how that will be possible ?

**Comment by planger** [comment, marked as answer]
Typically a Theia server is designed to run per user workspace. To manage workspaces, you would typically need a workspace management system, such as Eclipse Che or gitpod or plain kubernetes and start a dedicated pod with a Theia server for each workspace.
